src/main/java/climateControl/api/ClimateControlSettings.java

```diff
@@ -279,7 +279,7 @@ public void accept(Boolean accepted) {
         .booleanSetting(interveneInHighlandsName, false, "impose Climate Control generation on Highlands world types");
 
     public final Mutable<Boolean> noBoPSubBiomes = climateControlCategory
-        .booleanSetting(noBoPSubBiomesName, true, "suppress Bop sub-biome generation");
+        .booleanSetting(noBoPSubBiomesName, false, "suppress Bop sub-biome generation");
```

Comment:
Why flip this default?

Reply:
This option was true due to a bug in generating subbiomes in the deep ocean. 

In this PR, the generation of subbiomes in the deep ocean is prohibited, and therefore the bug due to which this option was true is no longer present, and there is no reason not to generate subbiomes